distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -0,0 +1,31 @@
+= Distributed Data Framework Documentation
+include::${project.build.directory}/doc-contents/_contents/config.adoc[]
+
+= Introduction
+
+== About ${branding}
+include::{adoc-include}/_introduction/introduction-contents.adoc[]
+
+include::{adoc-include}/_introduction/applications-contents.adoc[]
+
+== Documentation Guide
+
+include::{adoc-include}/_introduction/documentation-notes-contents.adoc[]
+
+<<<
+
+= ${branding} Core Concepts
+
```

Comment:
@pklinef @michael This section needs overhaul. I'd like to start with top level bullet points of the sections that _should_ be in the "Core Concepts" section. 

- Metadata
- Federation